components/wso2is7.key.manager/src/main/java/org/wso2/is7/client/model/TenantResponseModel.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com)
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.is7.client.model;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+/**
+ * Represents the tenant response model for WSO2 Identity Server 7.
+**/
+public class TenantResponseModel  {
```

Comment:
avoid model in the name. segregate the logic into simple model objects which  adhere to a single responsibility principle .

Reply:
renamed the classes